src/Integration/Suppression/SonarQubeIssuesProvider.cs

```diff
@@ -75,7 +75,22 @@ public void Dispose()
         public IEnumerable<SonarQubeIssue> GetSuppressedIssues(string projectGuid, string filePath)
         {
             // TODO: Block the call while the cache is being built + handle multi-threading
-            return this.cachedSuppressedIssues.Where(x => x.FilePath == filePath && x.ModuleKey == BuildModuleKey(projectGuid));
+
+            // HACK: ensure we've got data to enable end to end testing
```

Comment:
I don't think 'HACK' is picked up by analyzers. Perhaps add a TODO to make it easier to discover, just in case?

Reply:
Changed _HACK_ to _TODO_